website/content/docs/job-specification/upstreams.mdx

```diff
@@ -86,6 +89,22 @@ job "countdash" {
   local Consul datacenter.
 - `local_bind_address` - `(string: "")` - The address the proxy will receive
   connections for the upstream on.
+- `mesh_gateway` <code>([mesh_gateway][mesh_gateway_param]: nil)</code> - Configures the mesh gateway
+  behavior for connecting to this upstream.
+
+### `mesh_gateway` Parameters
+
+- `mode` `(string: "default")` - The mode of operation in which to use [Connect Mesh Gateways][mesh_gateways]
```

Comment:
Wow good catch! The source of truth here is "whatever consul does"
https://www.consul.io/docs/connect/config-entries/service-defaults#meshgateway
which is accepting, "", "none", "local", and "remote". We should have the empty string pass through, rather than canonicalize it to one of the modes, so that folks can use a Consul service-default config entry to provide a default behavior of their choosing. I'll fix the docs and the Canonicalize method.